interface/batchcom/batchPhoneList.php

```diff
@@ -6,42 +6,45 @@
 // create a list for phone calls
 // menu for fields could be added in the future
 
+include_once("../globals.php");
+use OpenEMR\Core\Header;
+
 ?>
 <html>
 <head>
-<?php html_header_show();?>
-<link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css">
-<link rel="stylesheet" href="batchcom.css" type="text/css">
```

Comment:
Place a title element so it shows up in the tab:
``````
<title><?php echo xlt("Phone Call List"); ?></title>